airflow/models/dagbag.py

```diff
@@ -727,3 +734,41 @@ def _sync_perm_for_dag(cls, dag: DAG, session: Session = NEW_SESSION):
 
         security_manager = ApplessAirflowSecurityManager(session=session)
         security_manager.sync_perm_for_dag(root_dag_id, dag.access_control)
+
+
+class DagPriorityParsingRequests(Base):
+    """Model to store the dag parsing requests that will be prioritized when parsing files."""
+
+    __tablename__ = "dag_priority_parsing_requests"
+
+    id = Column(String(40), primary_key=True)
```

Comment:
```suggestion
    id = Column(String(32), primary_key=True)
```

Since this is the length of a MD5…?